mkdocs.yml

```diff
@@ -16,6 +16,9 @@ plugins:
       fallback_to_build_date: true
 repo_url: https://github.com/chatterino/chatterino2
 edit_uri: ""
+markdown_extensions:
+  - toc:
+      permalink: true
```

Comment:
This adds anchors for quick url referencing.
![image](https://user-images.githubusercontent.com/12804673/96976061-b43df300-155e-11eb-92f5-28e5f45e9757.png)

Like this:
![image](https://user-images.githubusercontent.com/12804673/96976050-b1430280-155e-11eb-86d8-ddfc88e750fe.png)